miio/integrations/vacuum/roborock/vacuum.py

```diff
@@ -365,6 +369,41 @@ def map(self):
         # returns ['retry'] without internet
         return self.send("get_map_v1")
 
+    @command()
+    def get_multi_maps(self) -> MultiMapList:
```

Comment:
```suggestion
    def get_maps(self) -> MapList:
```
What do you think about renaming these?

Reply:
I have renamed all "multi_map" to "map"